test/e2e/run-multiple.sh

```diff
@@ -17,11 +17,19 @@ fi
 
 FAILED=()
 
+# Check if a save execution is passed to the script
+EXECUTION_FLAG=""
+
 for MANIFEST in "$@"; do
 	START=$SECONDS
 	echo "==> Running testnet: $MANIFEST"
 
-	if ! ./build/runner -f "$MANIFEST"; then
+	if [ -n "${SAVE_EXECUTION+set}" ]; then
+		EXECUTION_FLAG="--execution"
+		echo "==> Execution flag detected, saving files after run"
+	fi
+
+	if ! ./build/runner -f "$MANIFEST" "$EXECUTION_FLAG"; then
```

Comment:
Notice that we still dumps the manifest and logs to stdout. But I think we need that for the CI, right?

Maybe if `SAVE_EXECUTION` is set, we could skip the dumping?

Reply:
I left it there because on CI we need the dumps so we can download the logs from the Github Actions. This variable was introduced to run this locally. I'm not sure if we can get the assets from the Github Action node runners so specifying this variable would not make sense, this is more for local testing for now.

Reply:
My suggestion is that the code inside this `if` is kind of irrelevant when `-e` is passed. Because it will print the manifest, dump all logs, then cleanup. The former two are already performed by the new code, storing everything inside the `networks_executions/` dir.